pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2066,6 +2066,27 @@ public class ServiceConfiguration implements PulsarConfiguration {
     )
     private int loadBalancerBrokerThresholdShedderPercentage = 10;
 
+    @FieldContext(
+            dynamic = true,
+            category = CATEGORY_LOAD_BALANCER,
+            doc = "In the UniformLoadShedder strategy, the minimum msg rate that triggers unload."
+    )
+    private int minUnloadMessage = 1000;
```

Comment:
name should respect the description in doc

Reply:
OK

Reply:
fixed. PTAL,thanks! @aloyszhang